.github/scripts/git-secrets

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# ParlAI's modifications from original:
+# - automatically calls aws-provider, and hardcodes AWS parameters
```

Comment:
thanks for including these comments - could you also note in the code explicitly where the changes are?